packages/transformers/jsonld/src/JSONLDTransformer.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import json5 from 'json5';
+import { isURL, urlJoin } from '@parcel/utils';
+
+// A list of all attributes in a schema that may produce a dependency
+// Based on https://schema.org/ImageObject
+// Section "Instances of ImageObject may appear as values for the following properties"
+const SCHEMA_ATTRS = [
+  'logo',
+  'photo',
+  'image',
+  'thumbnail',
+  'screenshot',
+  'primaryImageOfPage',
+  'embedUrl',
+  'thumbnailUrl',
+  'video',
+  'contentUrl',
+  'url'
+];
+
+export default new Transformer({
+  async transform({ asset, config, logger, options, resolve }) {
+    // allowing any recieved jsonld to be in json5 format
```

Comment:
from which of the available object (config or options?) would I be able to get the `publicURL`? Whenever I try to reference it like `option.publicURL` I get a compiler error from flow

Reply:
as far as I know publicUrl should no longer be directly used in a transformer as we do url replacement all the way at the end after packaging. We just insert references in transformers